src/api.js

```diff
@@ -163,16 +165,19 @@ api.declare({
       msg.organization = sanitizeGitHubField(body.repository.owner.login),
       msg.action = body.action;
       msg.details = getPullRequestDetails(body);
+      msg.installationID = body.installation.id;
```

Comment:
Is that necessary?  It sounds like a lot of work, to not use pulse to separate webhooks from handlers.

I don't think the installation ID's are necessarily secret.  We *do* need to be careful that they cannot be forged, though.  I'll check that while reviewing.